packages/styles/scss/components/footer/_footer-logo.scss

```
@@ -64,12 +64,19 @@
   }
 
   :host(#{$c4d-prefix}-footer-logo[size='micro']) {
-    align-self: center;
-    margin: 0;
+    position: relative;
 
-    @include breakpoint-between(md, lg) {
+    .#{$c4d-prefix}--footer-logo__link {
       position: absolute;
-      bottom: to-rem(58.5px);
+      top: -37px;
```

```suggestion
      top: to-rem(-37px);
```
We could use the `to-rem` function here too

---

I tried this but the mixin did not like the negative value so I left it like this.